addons/mail/static/src/components/discuss/tests/discuss_tests.js

```diff
@@ -3063,8 +3063,18 @@ QUnit.test('do not post message on mailing channel with "Enter" keyboard shortcu
 
 QUnit.test('rendering of inbox message', async function (assert) {
     // AKU TODO: kinda message specific test
-    assert.expect(7);
+    assert.expect(8);
```

Comment:
Don't update this test, it's good that you add another one specific for checking the presence of the button.